pages/index.js

```diff
@@ -3,33 +3,70 @@ import Form from "@/components/Form/Form.js";
 import ShoppingItem from "@/components/ShoppingItem/ShoppingItem.js";
 import { useState } from "react";
 import { categories } from "@/lib/categoriesData";
+import Image from "next/image";
+
 export default function ShoppingItemsList({
   shoppingItemsWithCategoryColor,
+  backgroundColor,
+  shoppingItems,
   onAddItem,
   onEditItem,
   onDeleteItem,
+  onToggleIsPurchased,
+  purchasedItems,
+  unpurchasedItems,
 }) {
   const [mode, setMode] = useState("");
```

Comment:
I recommend to set this state to either `"add"`, `"edit"` or `"default"` (instead of an empty string). This way, you can easily check if the mode is neither `“add”` nor `“edit”` by using: `mode === "default"`.

```suggestion
  const [mode, setMode] = useState("default");
```

Reply:
Done.